src/librustdoc/lib.rs

```diff
@@ -704,6 +706,14 @@ fn opts() -> Vec<RustcOptGroup> {
             "removed, see issue #44136 <https://github.com/rust-lang/rust/issues/44136> for more information",
             "[rust]",
         ),
+        opt(
+            Unstable,
+            Flag,
+            "",
+            "generate-macro-expansion",
+            "Add possibility to expand macros in the HTML source code pages",
+            "",
+        ),
```

Comment:
All of the flags at the end of the list are deprecated or removed.  This flag should be moved before the "deprecated / removed options" comment.

Reply:
Shame on me: we even have a code comment to mark where the removed flags start...